frontend/public/components/persistent-volume.jsx

```diff
@@ -1,26 +1,50 @@
 import * as React from 'react';
 
 import { ColHead, DetailsPage, List, ListHeader, ListPage } from './factory';
-import { Kebab, LabelList, navFactory, ResourceKebab, SectionHeading, ResourceLink, ResourceSummary, Timestamp } from './utils';
+import { Kebab, LabelList, navFactory, ResourceKebab, SectionHeading, ResourceLink, ResourceSummary, Timestamp, StatusIcon } from './utils';
+
+const pvPhase = pv => pv.status.phase;
 
 const { common } = Kebab.factory;
 const menuActions = [...common];
 
+const PVStatus = ({pv}) => {
+  const phase = pvPhase(pv);
+  return <StatusIcon status={phase} />;
+};
+
 const Header = props => <ListHeader>
-  <ColHead {...props} className="col-sm-4 col-xs-6" sortField="metadata.name">Name</ColHead>
-  <ColHead {...props} className="col-sm-4 col-xs-6" sortField="metadata.labels">Labels</ColHead>
-  <ColHead {...props} className="col-sm-4 hidden-xs" sortField="metadata.creationTimestamp">Created</ColHead>
+  <ColHead {...props} className="col-sm-2 col-xs-6" sortField="metadata.name">Name</ColHead>
+  <ColHead {...props} className="col-sm-2 hidden-xs" sortField="status.phase">Status</ColHead>
+  <ColHead {...props} className="col-sm-2 hidden-xs" sortField="spec.claimRef.name">Persistent Volume Claim</ColHead>
+  <ColHead {...props} className="col-sm-2 hidden-xs" sortField="spec.capacity.storage">Capacity</ColHead>
+  <ColHead {...props} className="col-sm-2 col-xs-6" sortField="metadata.labels">Labels</ColHead>
+  <ColHead {...props} className="col-sm-2 hidden-xs" sortField="metadata.creationTimestamp">Created</ColHead>
 </ListHeader>;
 
 const kind = 'PersistentVolume';
+
 const Row = ({obj}) => <div className="row co-resource-list__item">
-  <div className="col-sm-4 col-xs-6">
+  <div className="col-sm-2 col-xs-6">
     <ResourceLink kind={kind} name={obj.metadata.name} namespace={obj.metadata.namespace} title={obj.metadata.name} />
   </div>
-  <div className="col-sm-4 col-xs-6">
+  <div className="col-sm-2 hidden-xs">
+    {<PVStatus pv={obj} />}
+  </div>
+  <div className="col-sm-2 hidden-xs">
+    {!_.isEmpty(obj.spec.claimRef) ?
+      <ResourceLink kind="PersistentVolumeClaim" name={obj.spec.claimRef.name} namespace={obj.spec.claimRef.namespace} title={obj.spec.claimRef.name} />
+      :
+      <div className="text-muted" key="0">No Persistent Volume Claim</div>
+    }
+  </div>
+  <div className="col-sm-2 hidden-xs">
+    {obj.spec.capacity.storage}
```

Comment:
Will this always be set? We might use `_.get` here as well to be safe

Reply:
i think it is better use _.get